include/solvers/heat_transfer.h

```diff
@@ -13,6 +13,14 @@
  *
  * ---------------------------------------------------------------------
 
+
+ *
+ * Implementation of heat transfer as an auxiliary physics.
+ * This heat equation is weakly coupled to the velocity field.
+ * Equation solved:
+ * rho * Cp * (dT/dt + u.gradT) = k div(gradT) + nu/rho * (gradu : gradu)
```

Comment:
```suggestion
 * \f$ \rho * C_p * (\frac{dT}{dt} + u \cdot \nabla T) = k \nabla^2T + \tau : \nabla u \f$
```
You should write the documentation using LaTeX format.